src/server/verification/verificationAPI.js

```diff
@@ -269,7 +269,7 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
     wrapAsync(async (req, res, next) => {
       const log = req.log.child({ from: 'verificationAPI - verify/email' })
       const { user, body } = req
-      const verificationData: { code: string } = body.verificationData
+      const verificationData: { code: integer } = body.verificationData
```

Comment:
`code` is no being used, why the destructure anyway?
 
---
as a note about typings:

Similar to above, despite it's composed only by numbers it's treated as a string.

It was a forced setup, so when calling https://github.com/GoodDollar/GoodServer/pull/58/files#diff-cafe901ba068ae37e72ff89b469d55dbR235 it's stored in GunDB as `string`

Then the data sent by the ÐApp is a string as well: https://github.com/GoodDollar/GoodDAPP/blob/b6fb38bcbe1ab7adc4531b2a35d1117d9da2e4eb/src/components/signup/SmsForm.js#L70 (this is from the SMS/OTP flow)